index.js

```diff
@@ -214,6 +239,54 @@ const lookUpCommand = (message) => {
     });
 };
 
+/*
+
+  const topFiveCommand = (message) => {};
+    The user uses this command to select up to 5 cards  from their inventory to display
+    any user can use the lookup command to look at another users top 5 cards, but only the owner of cards can select which cards go into their top 5
+    
+    To input a card to your top 5, the command must be entered as !showcase blue-eyes white dragon 
+
+
+
+*/
+
+const removeCardCommand = (message) => {
+  const removeThisCard = message.content.split(" ").slice(1).join(" ");
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventory = userData[userIdentification].inventory;
+    const inventoryRef = ref.child(message.author.id).child("inventory");
+
+    const cardIDToRemove = Object.keys(userInventory).find(
+      (key) =>
+        userInventory[key].name.toLowerCase() === removeThisCard.toLowerCase()
+    );
+    console.log(cardIDToRemove);
+
+    if (cardIDToRemove) {
+      const deletedCardName = userInventory[cardIDToRemove].name;
```

Comment:
shouldn't this be the same as the `removeThisCard` above? we can just re-use that instead of making a new variable

Reply:
The card name provided by the API is more careful with punctuation than the user input for the card names, so I thought it would be better to pull it from there. Mostly, just for aesthetics.

Reply:
The card name provided by the API is more careful with punctuation than the user input for the card names, so I thought it would be better to pull it from there.

Reply:
good point, really its just the casing that will be different, but good to use the real casing